src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -115,7 +115,7 @@ public function __construct(ClassMetadataFactoryInterface $classMetadataFactory
             throw new InvalidArgumentException(sprintf('The "%s" given in the default context is not callable.', self::MAX_DEPTH_HANDLER));
         }
 
-        $this->defaultContext[self::EXCLUDE_FROM_CACHE_KEY] = [self::CIRCULAR_REFERENCE_LIMIT_COUNTERS];
+        $this->defaultContext[self::EXCLUDE_FROM_CACHE_KEY] = [self::CIRCULAR_REFERENCE_LIMIT_COUNTERS, self::OBJECT_TO_POPULATE];
```

Comment:
I wondered about this while merging up: this means ppl must remember to list `OBJECT_TO_POPULATE` when they override `EXCLUDE_FROM_CACHE_KEY`. Is this wanted? Or do we prefer to *always* exclude `OBJECT_TO_POPULATE`?

Reply:
You're right, maybe we want to always exclude this key.
Why not doing it for `CIRCULAR_REFERENCE_LIMIT_COUNTERS` then?
Maybe a default context for this key has no sense at all.

Reply:
Actually, we should merge https://github.com/symfony/symfony/pull/36340 first then this one.

Reply:
We must a default context for this key to let the user exclude his own keys that may affect performance if the associated values aren't serializable.

Reply:
The problem is in this line though:
https://github.com/symfony/symfony/blob/bfe6b6fb41b677a35e7fbc89fcc7d9382be0c41b/src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php#L575
If the user wants to use the `EXCLUDE_FROM_CACHE_KEY` in the context (*not* the default one), then the *default* context values are overridden.
Which is not really a wanted behavior, doesn't it?

Reply:
I think it is. The context system works like that everywhere in the component (the local context overrides the default one). Let's not add exceptions.